screens/map-screen/map-screen.component.jsx

```diff
@@ -1,12 +1,35 @@
 import React from "react";
-import MapView from "react-native-maps";
-import { StyleSheet, View, Dimensions } from "react-native";
+import { StyleSheet, View } from "react-native";
+
+import Map from "../../components/map/map.component";
+import FetchCampusLocation from "../../components/fetchCampusLocation/FetchCampusLocation.component";
 
 class MapScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      region: {
+        latitude: 45.495869,
```

Comment:
Same as above

Reply:
this is where the initial location is set, which we all agreed that it'll be the current location that will be fetched using your pr